src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -27,6 +28,17 @@ public class PackageItemListViewModel : INotifyPropertyChanged
 
         public VersionRange AllowedVersions { get; set; }
 
+        private ImageMoniker _prefixReservedIndicatorMoniker;
+        public ImageMoniker PrefixReservedIndicatorMoniker
```

Comment:
i am not sure if i understand this - the ImageMoniker will always be the same image right? Why does it need to trigger PropertyChanged event? We only need to toggle visibility of the image based on PrefixReserved .